python/lsst/ctrl/bps/wms/htcondor/htcondor_service.py

```diff
@@ -81,6 +82,83 @@ def submit(self, workflow):
             htc_submit_dag(workflow.dag, dict())
             workflow.run_id = workflow.dag.run_id
 
+    def list_submitted_jobs(self, wms_id=None, user=None, require_bps=True, pass_thru=None):
+        """Query WMS for list of submitted WMS workflows/jobs.  This should
+        be a quick lookup function to create list of jobs ids for cancel
+        and other functions.
```

Comment:
Can we "move out" the second sentence from the summary to the extended description?